modules/webbrowser/include/modules/webbrowser/processors/webbrowserprocessor.h

```diff
@@ -94,75 +74,41 @@ namespace inviwo {
  * </script>
  * \endcode
  *
- * ### Inports
- *   * __background__ Background to render web page ontop of.
- *
- * ### Outports
- *   * __webpage__ GUI elements rendered by web browser.
- *
- * ### Properties
- *   * __URL__ Link to webpage, online or file path.
- *   * __Reload__ Fetch page again.
- *   * __Property__ Type of property to add.
- *   * __Html id__ Identifier of html element to synchronize. Not allowed to contain dots, spaces
- * etc.
- *   * __Add property__ Create a property of selected type and identifier. Start to synchronize
- * against loaded webpage.
  */
 
 /**
  * \brief Render webpage into the color and picking layers (OpenGL).
  */
-#include <warn/push>
-#include <warn/ignore/dll-interface-base>  // Fine if dependent libs use the same CEF lib binaries
-#include <warn/ignore/extra-semi>  // Due to IMPLEMENT_REFCOUNTING, remove when upgrading CEF
-class IVW_MODULE_WEBBROWSER_API WebBrowserProcessor : public Processor, public CefLoadHandler {
+class IVW_MODULE_WEBBROWSER_API WebBrowserProcessor : public Processor, public ProgressBarOwner {
 public:
-    WebBrowserProcessor(InviwoApplication* app);
-    virtual ~WebBrowserProcessor();
+    explicit WebBrowserProcessor(InviwoApplication* app);
```

Comment:
would be nice it this processor also had some import, maybe a data frame or json object that automatically got exposed in the webpage

Reply:
see https://github.com/inviwo/modules/pull/234

Reply:
I would prefer to make this more generic. I think we we add support for a string property for the html and a data frame / json port, There would be much less need for custom processor for each web thing

Reply:
Right now this processor is quite useless...

Reply:
I agree. This processor exists now purely for legacy reasons.